source/getting-started/models.md

```diff
@@ -0,0 +1,66 @@
+Now, let's add a list of available rentals to the index template.  We know that rentals will not be static.  Eventually, users will be able to add, update or delete them.  For this reason, we will need a _rentals_ model to save information about the rentals. However, to keep things simple at first, our model is just going to be a hard-coded array of JavaScript objects. Later, we'll switch to using Ember Data, so that we can let users create and modify their own rentals.
+
+Here's what our homepage will look like when we're done:
+
+![super rentals homepage with rentals list](../../images/models/super-rentals-index-with-list.png)
+
+Since the route handler is responsible for loading model data, that's where we'll start. Let's open `app/routes/index.js` and add our hard-coded data as the model:
+
+```app/routes/index.js
+import Ember from 'ember';
+
+var rentals = [{
+  id: 1,
+  owner: "Veruca Salt",
+  city: "San Francisco",
+  type: "Estate",
+  bedrooms: 15,
+  image: "https://upload.wikimedia.org/wikipedia/commons/c/cb/Crane_estate_(5).jpg"
+}, {
+  id: 2,
+  owner: "Mike TV",
+  city: "Seattle",
+  type: "Condo",
+  bedrooms: 1,
+  image: "https://upload.wikimedia.org/wikipedia/commons/0/0e/Alfonso_13_Highrise_Tegucigalpa.jpg"
+}, {
+  id: 3,
+  owner: "Violet Beauregarde",
+  city: "Portland",
+  type: "Apartment",
+  bedrooms: 3,
+  image: "https://upload.wikimedia.org/wikipedia/commons/f/f7/Wheeldon_Apartment_Building_-_Portland_Oregon.jpg"
+}];
+
+export default Ember.Route.extend({
+  model() {
+    return rentals;
+  },
+});
+```
+
+If you haven't used the ES6 shorthand method definition syntax before, `model()` is the same as writing `model: function()`.
+
+When we add a method to a class as part of using the Ember framework, we often call it a `hook`. So here, we've added the model hook to our `index` route handler.
```

Comment:
is this definition sufficient? I'm uncertain it's clear that `model` is a method that Ember calls when you go to a route